src/compiler/parser.ts

```diff
@@ -440,7 +440,26 @@ namespace ts {
 
     /* @internal */
     export function parseIsolatedJSDocComment(content: string, start?: number, length?: number) {
-        return Parser.JSDocParser.parseIsolatedJSDocComment(content, start, length);
+        const result = Parser.JSDocParser.parseIsolatedJSDocComment(content, start, length);
+        if (result && result.jsDocComment) {
+            // because the jsDocComment was parsed out of the source file, it might
+            // not be covered by the fixupParentReferences.
+            let parentNode: Node = result.jsDocComment;

```

Comment:
can you just call `fixupParentReferences` here and pass `result.jsDocComment` as a parameter?